tests/ctst/common/common.ts

```diff
@@ -1,12 +1,36 @@
 import fs from 'fs';
 import path from 'path';
-import { Given, setDefaultTimeout } from '@cucumber/cucumber';
+import { Given, setDefaultTimeout, Then } from '@cucumber/cucumber';
 import { Constants, S3, Utils } from 'cli-testing';
 import Zenko from 'world/Zenko';
-import { extractPropertyFromResults } from './utils';
+import { extractPropertyFromResults, safeJsonParse } from './utils';
+import assert from 'assert';
 
 setDefaultTimeout(Constants.DEFAULT_TIMEOUT);
 
+async function addMultipleObjects(this: Zenko, numberObjects: number,
+    objectName: string, sizeBytes: number, userMD?: string) {
+    for (let i = 1; i <= numberObjects; i++) { 
+        this.addToSaved('objectName', `${objectName}-${i}` || Utils.randomString());
+        const objectPath = path.join(__dirname, this.getSaved<string>('objectName'));
```

Comment:
Usually we create such files in the `/tmp/` directory to ensure write access, depending on how the Dockerfile evolves, it may not work anymore at some point